core/store/migrations/0040_add_run_id_to_pipeline_runs.go

```diff
@@ -0,0 +1,42 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up40 = `
+	CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
This might cause problems for node operators, since many run chainlink as a database user that does not have CREATE EXTENSION permissions.

Is there any way we can avoid using it?

Reply:
We could if we don't make the field nullable. Setting it to `NOT NULL` requires us to migrate old data